src/qt/aboutdialog.cpp

```diff
@@ -11,6 +13,12 @@ AboutDialog::AboutDialog(QWidget *parent) :
     ui->copyrightLabel->setText("Copyright 2009-2024 The Bitcoin/Peercoin/Black-Coin/Gridcoin developers");
 
     resize(GRC::ScaleSize(this, width(), height()));
+
+    if (!fTestNet) {
+        connect(ui->versionInfoButton, &QAbstractButton::pressed, this, [this]() { handlePressVersionInfoButton(); });
+    } else {
+        ui->versionInfoButton->hide();
```

Comment:
nit: Would be nicer to disable the button with a message instead of hiding it.
```suggestion
        ui->versionInfoButton->setDisabled(true);
        ui->versionInfoButton->setToolTip(tr("Version information is not available on testnet."));
```

Reply:
Seems equivalent in this case since the only way you can get to that button is this one screen, but ok.